### ISSUE TITLE
Is it possible to have "dashboard-banner-logo-title" in multiple line align center or some margin? 

### ISSUE THREAD

**Issue by testinggithub1222** [emacs-dashboard/emacs-dashboard#121]
I have tried to set **(setq dashboard-banner-logo-title "half text \n another half")** but the problem is that, the first half text is align center inside Emacs while another half is showing on the left of the buffer in emacs. I have tried to add "\t\t\t" to indent the text to center but when switching to another pc, it shows in a difference location. It might be because computer screen resolution is not the same.

I want it to format as following:
```
                           Center Align
                   Center Align for Longer Text
                   Center Align for Longer Text
                            Center for 
                            

Or 
                           Center Align
                   Center Align for Longer Text
                   Center Align for Longer Text
                   Center for 
                   Center for 
                   Center for 
```

**Comment by seagle0128** [comment]
Do you want this?
<img width="2032" alt="image" src="https://user-images.githubusercontent.com/140797/58311488-ad77f180-7e3b-11e9-8919-db36367e3fee.png">

**Comment by jcs090218** [comment]
This is great if we have this implemented!

**Comment by jjibagar** [comment]
@seagle0128  and how do you do the horizontal lines?

**Comment by seagle0128** [comment]
@jjibagar @jcs090218 See https://github.com/seagle0128/.emacs.d/blob/master/lisp/init-dashboard.el#L91. I think this feature has already been integrated into `emacs-dashboard`.

**Comment by jcs090218** [comment]
Yes, it's implemented. Therefore, closing this issue! Thanks! :D
